docs/modules/ROOT/pages/utilities.adoc

```diff
@@ -0,0 +1,43 @@
+= Utilities
+
+The OpenZeppelin Stylus Contracts provide a ton of useful utilities that you can use in your project.
+For a complete list, check out the https://docs.rs/openzeppelin_stylus/utils/index.html[API Reference].
```

Comment:
Shouldn't we pin to a specific version? In this case `0.1.0-rc`, otherwise when we publish new releases the docs for the old version will point to the new api docs